src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java

```diff
@@ -0,0 +1,187 @@
+package com.wikia.webdriver.testcases.Anonymization;
+
+import com.wikia.webdriver.common.contentpatterns.PageContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.api.ArticleContent;
+import com.wikia.webdriver.common.core.api.UserRegistration;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.SignUpUser;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.global_navitagtion.NavigationBar;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.minieditor.MiniEditorComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.UserProfilePage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.DetachedSignInPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.SignInPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.messagewall.MessageWall;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.ArticleHistoryPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.anonymization.SpecialAnonymizationUserPage;
+
+import org.joda.time.DateTime;
+import org.testng.annotations.Test;
+
+import java.time.LocalDate;
+
+@SuppressWarnings("SpellCheckingInspection")
+@InBrowser(browser = Browser.CHROME)
+@Execute(onWikia = "communitytest")
+@Test(groups = "anonymization")
+public class AnonymizationTests extends NewTestTemplate {
+
+
+  private static final String ERROR_MESSAGE =
+      "We don't recognize these credentials. Try again or register a new account.";
+
+  private SignInPage openLoginModalOnDesktop() {
+    return new DetachedSignInPage(new NavigationBar().clickOnSignIn());
+  }
+
+  @Test
+  public void anonymizedUserCannotLogin() {
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    String qanon = "QAanon" + timestamp;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "QAanon" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)